index.js

```diff
@@ -200,7 +219,6 @@ export const isDynamicPattern = normalizeArgumentsSync(
 export const generateGlobTasks = normalizeArguments(generateTasks);
 export const generateGlobTasksSync = normalizeArgumentsSync(generateTasksSync);
 
-export {
-	isGitIgnored,
-	isGitIgnoredSync,
-} from './gitignore.js';
+export {isIgnored, isIgnoredSync} from './ignore.js';
```

Comment:
I'm not sure if we want to expose these, if we do, maybe we need a better name, `isIgnoredByIgnoreFiles`?